src/content/guides/fine-tuning-upgrade-disruption-on-aws/index.md

```diff
@@ -1,123 +1,136 @@
-  name: jni9x
-  namespace: default
-spec:
- ....
-
+ ...
 ```
 
-In this other example, we set value to `0.10` to roll a maximum of 10% of nodes per single batch.
+In this second example, we set the value to `0.1` to roll a maximum of 10 percent of nodes of a node pool per single batch.
 
 ```yaml
-
 apiVersion: infrastructure.giantswarm.io/v1alpha2
 kind: AWSCluster
 metadata:
-  annotations:
-    alpha.aws.giantswarm.io/update-max-batch-size: "0.10"
-  labels:
-    giantswarm.io/cluster: jni9x
-    giantswarm.io/organization: giantswarm
-    release.giantswarm.io/version: 12.7.0
   name: jni9x
-  namespace: default
-spec:
- ....
+  annotations:
+    alpha.aws.giantswarm.io/update-max-batch-size: "0.1"
+ ...
+```
+
+## Pause Time {#pause-time}
+
+After updating a batch of worker nodes, a pause time is applied. This time effectively allows Kubernetes to schedule workloads on the new worker nodes.
+
+This setting maps to the [`PauseTime`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-attribute-updatepolicy.html#cfn-attributes-updatepolicy-rollingupdate-pausetime) property of the AWS CloudFormation update policy.
 
+**By default**, the pause time is set to 15 minutes.
+
+This value can be influenced via the annotation
+
+```nohighlight
+alpha.aws.giantswarm.io/update-pause-time
 ```
 
-## Pause Time
+Again, the setting can be defined on two levels:
 
-This parameter defines the amount of time that AWS CloudFormation pauses after making a change to a batch of instances to give those instances time to start software applications.
+- on the [`AWSCluster`](/reference/cp-k8s-api/awsclusters.infrastructure.giantswarm.io/) resource, to be applied as a default to **all node pools** of the cluster.
+- on the [`AWSMachineDeployment`](/reference/cp-k8s-api/awsmachinedeployments.infrastructure.giantswarm.io/) resource, to be effective for only **one node pool**. A value here will override any value specified on the `AWSCluster` level.
 
-This value can be set via annotation `alpha.aws.giantswarm.io/update-pause-time` on `AWSCluster` CR or `AWSMachineDeploymentCR`. The value set on `AWSCluster` will be used for a whole cluster. The value set on `AWSMachineDeploymentCR` will be only used for that specific NodePool and will override the value from `AWSCluster` in case it is defined.
+The value must be a string in the ISO 8601 duration format. Value examples:
 
-The `PauseTime` value needs to be in ISO 8601 duration format, e. g. "PT10M" for 10 minutes or "PT15S" for 15 seconds.
+- `PT10M`: 10 minutes
+- `PT15S`: 15 seconds
+- `PT1M30S`: 1 minute and 30 seconds
 
-The default value is `PT15M` (15 minutes).
+The maximum pause time is one hour (`PT1H`).
 
-### PauseTime examples
+### Examples {#pause-time-examples}
 
-In this example, we set the value to `PT1M30S` to pause for 1 minute and 30 seconds between each batch.
+In the first example, we set the value to `PT1M30S` to pause for one minute and 30 seconds between each batch.
```

Comment:
` one minute and 30 seconds` this feels weird to me that once we use the word and then we use the number, but if you feel its right then go with it

Reply:
I guess you're right. I was following the guideline "In scientific and technical writing, the prevailing style is to write out numbers under ten." here. But in direct combination, it would make more sense to use 1 instead of "one".